### ISSUE TITLE
Issue with opening website on Internet Explorer (Needs Info/testing)

### ISSUE THREAD

**Issue by ananyaarun** [publiclab/plots2#5096]
# Description
Although most technical user of the public lab website don't usually use Internet Explorer, I noticed that Opening publiclab.org on this browser has issues.
- The Get Involved and About us section doesn't open on clicking. 
- Only Donate and Store pages function as desired.

# Possible Solution
After more testing, if this is actually the case maybe a msg/alert can be given to the user saying best for "X browser Y version" or "recommended use X browser".
A lot of common users who want to post notes/wikis may still be using internet explorer , hence a msg like this would clear the confusion rather than discourage a user who isn't able to see anything on the website properly when opened.

**Comment by gauravano** [comment]
Confirmed by running the website on Internet Explorer via browserling. Dropdowns are not working. Thanks @ananyaarun for reporting.

**Comment by ananyaarun** [comment]
@gauravano , Can we add info regarding "Best in X browser" etc or "recommended in Y browser" in the readme or something to ensure no one opens it on IE to find nothing is working ?

**Comment by gauravano** [comment]
Either we can do the work to make website IE compatible or display message on detecting IE browser. @jywarren can we decide on basis of # of user who visit PL from IE, maybe? We have Google analytics setup right?

**Comment by ananyaarun** [comment]
Yes, If ratio of people using IE is less we can detach IE browser else something needs to be done to make it IE compatible as at the moment The website is pretty unusable on IE.
